net/siproxd/src/opnsense/mvc/app/models/OPNsense/Siproxd/General.xml

```diff
@@ -0,0 +1,182 @@
+	    <MinimumValue>1</MinimumValue>
+	    <MaximumValue>65535</MaximumValue>
+        </sip_listen_port>
+        <rtp_port_low type="IntegerField">
+            <default>7070</default>
+            <Required>Y</Required>
+            <MinimumValue>1</MinimumValue>
+            <MaximumValue>65535</MaximumValue>
+        </rtp_port_low>
+        <rtp_port_high type="IntegerField">
+            <default>7089</default>
+            <Required>Y</Required>
+            <MinimumValue>1</MinimumValue>
+            <MaximumValue>65535</MaximumValue>			
+        </rtp_port_high>
+        <rtp_timeout type="IntegerField">
+            <default>300</default>
+            <Required>Y</Required>
+            <MinimumValue>1</MinimumValue>
+            <MaximumValue>10000</MaximumValue>
+        </rtp_timeout>
+        <rtp_dscp type="IntegerField">
+            <default>46</default>
+            <Required>Y</Required>
+            <MinimumValue>0</MinimumValue>
+            <MaximumValue>64</MaximumValue>			
+        </rtp_dscp>
+        <sip_dscp type="IntegerField">
+            <default>0</default>
+            <Required>Y</Required>
+            <MinimumValue>0</MinimumValue>
+            <MaximumValue>64</MaximumValue>			
+        </sip_dscp>
+        <rtp_input_dejitter type="BooleanField">
+            <default>0</default>
+            <Required>N</Required>
+        </rtp_input_dejitter>
+        <rtp_output_dejitter type="BooleanField">
+            <default>0</default>
+            <Required>N</Required>
+        </rtp_output_dejitter>
+        <tcp_timeout type="IntegerField">
+            <default>600</default>
+            <Required>Y</Required>
+            <MinimumValue>1</MinimumValue>
+            <MaximumValue>10000</MaximumValue>			
+        </tcp_timeout>
+        <tcp_connect_timeout type="IntegerField">
+            <default>500</default>
+            <Required>Y</Required>
+            <MinimumValue>1</MinimumValue>
+            <MaximumValue>10000</MaximumValue>			
+        </tcp_connect_timeout>
+        <tcp_keepalive type="IntegerField">
+            <default>20</default>
+            <Required>Y</Required>
+            <MinimumValue>0</MinimumValue>
+            <MaximumValue>10000</MaximumValue>			
+        </tcp_keepalive>
+        <ua_string type="TextField">
+            <default></default>
+            <Required>N</Required>
+        </ua_string>
+        <use_rport type="OptionField">
+            <default>Option1</default>
+            <multiple>N</multiple>
+            <Required>Y</Required>
+                <OptionValues>
+                    <Option1 value="0">0 - do not add ;rport to via header</Option1>
+                    <Option2 value="1">1 - do add ;rport to INCOMING via header only</Option2>
+                    <Option3 value="2">2 - do add ;rport to OUTGOING via header only</Option3>
+                    <Option4 value="3">3 - do add ;rport to OUTGOING and INCOMING via headers</Option4>
```

Comment:
afaik threre is no value attribute. the tag is the value and the string inbetween is the visible string

Reply:
Yes, this requires patching:
https://github.com/opnsense/core/commit/c7c1c55c8e4c2e72c84ebd6e9212c4faa25ef9ee#commitcomment-24437550